src/Window.hpp

```diff
@@ -14,11 +14,19 @@ class Window {
 	Window(int width, int height, const char* title);
 	~Window();
 
+	void set_gl_version(unsigned major, unsigned minor);
```

Comment:
Note snake case name. I prefer to use `snake_case_gl_version` instead of `camelCaseGLVersion`

Reply:
However, we may want to change that for the sake of consistency